### DISCUSSION TITLE
PWA Support and example / tutroial

### DISCUSSION THREAD

**Discussion by raghav0622** [blitz-js/blitz#2179] [General]
Hello Everyone,

Can i support PWA in Blitz app, if yes how?

Thanks

**Comment by lksnmnn** [comment]
I think you should be able to configure the Blitz app as a PWA just like you would with any other Next.js application. Checkout https://github.com/shadowwalker/next-pwa or https://github.com/hanford/next-offline.

I have not added these to Blitz, but I'll need to do it later this month also, so let me know if these work ;)

**Comment by reymon359** [comment]
Hi there. I have configured my Blitz.js app with next-pwa and still doesn't behave as a PWA. 
Does someone know about a blitz.js PWA app example I can have a look at? 😄

**Reply by Kay2dan** [reply]
Upvoting as this will be useful to a lot of us.
